src/ExpressiveInstaller/config.php

```diff
@@ -39,7 +40,7 @@
     'questions' => [
         'container' => [
             'question'               => 'Which container do you want to use for dependency injection?',
-            'default'                => 3,
+            'default'                => 4,
```

Comment:
All the rest of them are in alphabetical order... shouldn't the default one be first? 🎩